functional-tests/affected-ftest/pom.xml

```diff
@@ -42,6 +42,9 @@
           <properties>
             <orderStrategy>affected</orderStrategy>
           </properties>
+          <includes>
+            <include>**/src/test/java/**/Test*.java</include>
```

Comment:
Changed to two includes to follow surefire notation and only allows `Test` at start or at end but not in the middle.